app/org/iatoki/judgels/sealtiel/services/impls/ClientServiceImpl.java

```diff
@@ -58,9 +54,14 @@ public void updateClient(String clientJid, String name, String userJid, String i
 
     @Override
     public List<Client> getAllClients() {
+        ImmutableList.Builder<Client> result = ImmutableList.builder();
```

Comment:
Instead of this you can use `Lists.transform` for brevity: `return Lists.transform(clientModels, model -> creteClientFromModel(model));`